cpp/src/io/parquet/reader_impl.cu

```diff
@@ -260,7 +260,7 @@ struct metadata : public FileMetaData {
       }
       row_count = 0;
       do {
-        selection.emplace_back(row_group, row_start);
+        selection.emplace_back(row_group, row_count);
```

Comment:
Should be row_start + row_count

Reply:
Thanks @OlivierNV, that change fixed the Python tests.